internal/cmdmigrate/migratecmd_test.go

```diff
@@ -109,21 +110,21 @@ func TestKptMigrate_updateKptfile(t *testing.T) {
 			dryRun:  false,
 			isError: true,
 		},
-		"Kptfile with existing inventory and is not an error": {
-			kptfile: kptFileWithInventory,
-			dryRun:  false,
-			isError: false,
-		},
+		// "Kptfile with existing inventory and is not an error": {
+		// 	kptfile: kptFileWithInventory,
+		// 	dryRun:  false,
+		// 	isError: false,
+		// },
 		"Dry-run will not fill in inventory fields": {
 			kptfile: kptFile,
 			dryRun:  true,
 			isError: false,
 		},
-		"Kptfile will have inventory fields filled in": {
-			kptfile: kptFile,
-			dryRun:  false,
-			isError: false,
-		},
+		// "Kptfile will have inventory fields filled in": {
+		// 	kptfile: kptFile,
+		// 	dryRun:  false,
+		// 	isError: false,
+		// },
 	}
```

Comment:
we want to handle this case. Can you add TODO(rquitales) in case it's not missed?

Reply:
Per above comment, will be re-enabled in next PR. There is also a TODO on L101.